api/src/main/java/ai/djl/ndarray/NDArray.java

```diff
@@ -4543,4 +4543,25 @@ default NDArray norm(int[] axes, boolean keepDims) {
      * @return the norm of this {@code NDArray}
      */
     NDArray norm(int ord, int[] axes, boolean keepDims);
+
+    /**
+     * Returns a one-hot {@code NDArray}.
```

Comment:
Can you add a bit more documentation? Many users who are newer to deep learning may not be familiar with this term. Also, add a `@see` link to `https://d2l.djl.ai/chapter_linear-networks/softmax-regression.html#classification-problems` which will add more context about how this operator is meaningful to deep learning